<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>PartEditBase</class>
 <widget class="QWidget" name="PartEditBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>-1</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string/>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>3</number>
      </property>
      <property name="spacing" >
       <number>4</number>
      </property>
      <item row="0" column="8" >
       <widget class="QSpinBox" name="minPitch" >
        <property name="maximum" >
         <number>127</number>
        </property>
       </widget>
      </item>
      <item row="1" column="8" >
       <widget class="QSpinBox" name="maxPitch" >
        <property name="maximum" >
         <number>127</number>
        </property>
        <property name="value" >
         <number>127</number>
        </property>
       </widget>
      </item>
      <item row="2" column="8" >
       <widget class="QLineEdit" name="longName" />
      </item>
      <item row="2" column="7" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Long Name:</string>
        </property>
        <property name="textFormat" >
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="7" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Short Name:</string>
        </property>
        <property name="textFormat" >
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="8" >
       <widget class="QLineEdit" name="shortName" />
      </item>
      <item row="1" column="7" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Max.Pitch:</string>
        </property>
        <property name="textFormat" >
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="7" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Min.Pitch:</string>
        </property>
        <property name="textFormat" >
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2" >
       <widget class="QComboBox" name="patch" />
      </item>
      <item row="1" column="2" colspan="2" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Midi Patch</string>
        </property>
        <property name="textFormat" >
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QCheckBox" name="solo" >
        <property name="text" >
         <string>Solo</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>59</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>59</width>
          <height>25</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" >
       <widget class="QCheckBox" name="mute" >
        <property name="text" >
         <string>Mute</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QSpinBox" name="channel" >
        <property name="maximum" >
         <number>16</number>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Channel</string>
        </property>
        <property name="textFormat" >
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2" >
       <widget class="QLineEdit" name="partName" />
      </item>
      <item row="3" column="1" >
       <widget class="QCheckBox" name="showPart" >
        <property name="text" >
         <string>Show</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4" colspan="3" >
       <widget class="QSlider" name="pan" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Pan</string>
        </property>
        <property name="maximum" >
         <number>127</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item rowspan="3" row="0" column="4" >
       <widget class="QSlider" name="volume" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>4</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Volume</string>
        </property>
        <property name="maximum" >
         <number>127</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item rowspan="3" row="0" column="5" >
       <widget class="QSlider" name="reverb" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>4</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Reverb Send</string>
        </property>
        <property name="maximum" >
         <number>127</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item rowspan="3" row="0" column="6" >
       <widget class="QSlider" name="chorus" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>4</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Chorus Send</string>
        </property>
        <property name="minimum" >
         <number>0</number>
        </property>
        <property name="value" >
         <number>0</number>
        </property>
        <property name="sliderPosition" >
         <number>0</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
